test/jdk/java/net/httpclient/whitebox/java.net.http/jdk/internal/net/http/SSLTubeTest.java

```diff
@@ -212,14 +216,18 @@ private void serverLoopback() {
                         is.close();
                         os.close();
                         serverSock.close();
+                        System.out.println("serverLooback exiting normally");
```

Comment:
Typo in the message, should have been `serverLoopback ...`. Same in 2 other places.

Reply:
will fix